tests/dynamic_data/publications/layer_geoserver_proxy/geoserver_proxy_test.py

```diff
@@ -0,0 +1,140 @@
+import copy
```

Comment:
I suggest to move the test to `tests/dynamic_data/publications/layer_wfst/auth_test.py`

Reply:
Moved and renamed.